docker/travis.tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+source ${docker_env}
+
+cd ${dagmc_build_dir}/DAGMC
+
+# clean out config test directory for next build
+git clean -dxf . 
+
+# Test DAGMC CMake configuration file
+cd ${dagmc_build_dir}/DAGMC/cmake/test_config
+cmake . -DDAGMC_ROOT=${dagmc_install_dir}
+make all test
+
+cd ${dagmc_build_dir}/bld
+
+# If this is not a pull request, get files needed for regression tests
+if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+  cd src/make_watertight/tests
+  wget ${MW_REG_TEST_MODELS_URL} -O mw_reg_test_files.tar.gz -o wget.out
```

Comment:
Instead of capturing the `wget` log files in `wget.out`, consider leaving the default and streaming them to `stderr`.  It is more standard to log to `stderr` and `stdout` in apps, so docker can forward these outputs to the logging driver.